Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM kong
+
+COPY kong/plugins/kubernetes-sidecar-injector /usr/local/share/lua/5.1/kong/plugins/kubernetes-sidecar-injector
+
+RUN chmod 664 /usr/local/share/lua/5.1/kong/plugins/kubernetes-sidecar-injector/*
```

Comment:
Shouldn't need to `chmod`. What failure did you have without it?

Reply:
I needed to locally but my filemasks are sometimes wonky. I'll try removing and seeing if Travis is happy with it

Reply:
managed to do this to my local setup at one point somehow
```
d????????? ? ? ? ?            ? .
d????????? ? ? ? ?            ? ..
-????????? ? ? ? ?            ? api.lua
-????????? ? ? ? ?            ? config.lua
```
but got it sorted I think